tests/git.zunit

```diff
@@ -1093,8 +1131,8 @@
     _fzf_complete_git 'git commit-cleanup '
 }
 
-@test 'Testing completion (alias): git commit --untracked-files **' {
-    run git config alias.commit-untracked 'commit --untracked-files'
+@test 'Testing completion (alias): git commit --untracked-files=**' {
```

Comment:
At a glance this fix seems correct but git does not accept optional argument through aliases. The preferable way is to change to some other option.